pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -511,6 +511,27 @@ def test_mangled(self):
         expected = pd.DataFrame({"a": [0, 0], "b": [1, 1]})
         tm.assert_frame_equal(result, expected)
 
+    def test_agg_namedtuple(self):
+        # GH34422
+        s = pd.Series([1, 1, 2, 2, 3, 3, 4, 5])
+        msg = "func is"
```

Comment:
Can this be more specific?

Reply:
Yes, Can be extend `msg = "func is expected but recieved NamedAgg"` in this test, while using tuple `msg = "func is expected but recieved tuple"` and while using list `msg = "func is expected but recieved list"`. Is this fine?